presto-main/src/test/java/io/prestosql/tests/ReportUnannotatedMethods.java

```diff
@@ -43,12 +43,16 @@ public void onBeforeClass(ITestClass testClass)
                 .collect(toImmutableList());
 
         if (!unannotatedMethods.isEmpty()) {
-            throw new RuntimeException(format(
-                    "Test class %s has methods which are public but not explicitly annotated. Are they missing @Test?%s",
+            // TestNG may or may not propagate listener's exception as test execution exception.
+            // Therefore, instead of throwing, we terminate the JVM.
+            System.err.println(format(
```

Comment:
Doing `println(format(` rather than `printf` because of https://github.com/airlift/airlift/issues/737